### DISCUSSION TITLE
fastestvpn: looks like servers and ips changed

### DISCUSSION THREAD

**Discussion by rsquarev** [qdm12/gluetun#1263] [Q&A] [answered]
looks like servers and ips changed for fastestvpn. Any plans to update this on the repository?

**Comment by qdm12** [comment, marked as answer]
Fixed thanks to your PR #1264 so now you can update servers data yourself following https://github.com/qdm12/gluetun/wiki/Updating-servers

I pushed 5070dbcf7f17ec25a5fdbf871f753e4043510867 which updates the servers data, so you can just pull the latest image once it's built as well :wink:
